dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Fedora's Static PSR-4, PSR-0, and classmap autoloader"
+HOMEPAGE="https://github.com/php-fedora/autoloader"
+SRC_URI="https://github.com/php-fedora/autoloader/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
The ebuild for fedora-autoloader-0.2.1 now has

```
KEYWORDS="alpha amd64 arm hppa ia64 ppc ppc64 ~s390 ~sh sparc x86"
```

Can you please add all of those to the 1.0.0 ebuild (as ~arch)?

Reply:
Yes I added it back 👍

Reply:
Can you make the ebuild unstable on all arches pleeeaase? Repoman will complain about adding a new version that is already stable:

```
RepoMan scours the neighborhood...
  KEYWORDS.stable [fatal]       1
   dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild added with stable keywords: alpha amd64 arm hppa ia64 ppc ppc64 sparc x86
```

Reply:
@orlitzky Yes this is why I removed them in the first place but I should have changed them to unstable, it is now done 🍺